src/puppetlabs/jdbc_util/core.clj

```diff
@@ -139,7 +140,9 @@
       (let [tail (.substring s (.length ^String match) (.length s))
             replaced (str/replace match #"(.*)\?$" (str "$1" replacement))]
         (str replaced tail))
-      (throw (IllegalArgumentException. (str "there are not " n " '?'s in the given string"))))))
+      (throw (IllegalArgumentException. (trun "There is not one '?' in the given string"
```

Comment:
This is prooobably a code bug but could not be. So I'm fine keeping it in but the singular case only happens when the string is supposed to contain a "?" and doesn't have any at all, so saying it doesn't have any at all gets the point across better.